src/core/formatCommaPositions.ts

```diff
@@ -1,7 +1,7 @@
 import type { CommaPosition } from 'src/types';
 import { maxLength } from 'src/utils';
 
-const WHITESPACE_REGEX = /^(\s+)/u;
+import { WHITESPACE_REGEX } from 'src/lexer/regexUtil';
```

Comment:
The new regex is not the same as the old one.

- old: `/^(\s+)/u`
- new: `/(\s+)/uy`